data-prepper-plugins/log-generator-source/src/main/java/com/amazon/dataprepper/plugins/source/loggenerator/ApacheLogFaker.java

```diff
@@ -93,10 +94,10 @@ private String randomDate() {
         return ZonedDateTime.now().minus(offset).format(dateTimeFormatter);
     }
 
-    public String generateRandomApacheLog() {
+    public String generateRandomExtendedApacheLog() {
         final int bytes = random.nextInt(1001) + 4000;
```

Comment:
Minor but it would be nice to move `random.nextInt(1001) + 4000;` into its own private method since it gets reused. A well named method would also make it more clear what this is for